### PR TITLE
Calendar header component

### PR DESCRIPTION
**Pull request by jankrzempek**
**GOAL**: Working (for now only presenting given data) components - displaying data and working on mobile and web
**TASK**: https://trello.com/c/PnN2MMOn/1730-calendar-station-header

**Example usage:**

```
 <CalendarStationHeader 
      stationName="White Plains, New York-Northern New Jersey-London" 
      profile="Tim Cain" 
      sensor="Government Data USEPA" 
      lastUpdate="18:00, Sep 1 2023"
      stationValues={{
        day: "Jun 12",
        value: 12,
        parameter: "PM2.5 µg/m"}}/>
```

### CODE REVIEW

**Review by smialko — COMMENTED** [review]
review tip

File: app/javascript/react/utils/media.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const sizes = {
+  mobile: "768px",
+  desktop: "769px",
+};
```
Comment (smialko) [inline app/javascript/react/utils/media.ts:4]: why? Could you add a comment or sth?  🙈

**Review by smialko — CHANGES_REQUESTED** [review]
Awesome code! Noticed a few differences from the designs so I'm requesting changes, please check. Thanks!

File: app/javascript/react/components/molecules/StationValueLabel/StationValueLabel.style.tsx [hunk 2]
```diff
@@ -10,34 +11,60 @@ const Container = styled.div`
     ),
     ${mint};
```
[the remark this answers is not in the document]
Comment (jankrzempek) [reply app/javascript/react/components/molecules/StationValueLabel/StationValueLabel.style.tsx:12]: That's interesting, how can we know which one to implement, or both? How to approach it?

File: app/javascript/react/components/molecules/StationValueLabel/StationValueLabel.style.tsx [hunk 3]
```diff
@@ -10,34 +11,60 @@ const Container = styled.div`
...
   padding: 34px 20px;
   width: 290px;
   height: 180px;
   border-radius: 10px;
+
+  @media ${media.mobile} {
+    flex-direction: column;
+    width: 111px;
+    height: 274px;
+    border-radius: 0px;
```
[the remark this answers is not in the document]
Comment (jankrzempek) [reply app/javascript/react/components/molecules/StationValueLabel/StationValueLabel.style.tsx:23]: One design says it should be, and the second says it should not. You right... asking Iwona now

File: app/javascript/react/components/molecules/CalendarStationHeader/CalendarStationHeader.style.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  font-weight: 600;
+  display: inline;
+`;
+
+const Description = styled(H4)`
+  padding-bottom: 10px;
+
+  @media ${media.mobile} {
+    font-size: 12px;
+    font-weight: 500;
```
[the remark this answers is not in the document]
Comment (jankrzempek) [reply app/javascript/react/components/molecules/CalendarStationHeader/CalendarStationHeader.style.tsx:36]: Zrzut ekranu 2024-01-15 o 10 45 53
Straight from designs

File: app/javascript/react/components/molecules/CalendarStationHeader/CalendarStationHeader.style.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+  @media ${media.mobile} {
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 500;
+  }
+`;
+
+const UpdateLabel = styled(H4)`
+  @media ${media.desktop} {
+    text-transform: uppercase;
```
[the remark this answers is not in the document]
Comment (jankrzempek) [reply app/javascript/react/components/molecules/CalendarStationHeader/CalendarStationHeader.style.tsx:65]: Thx, done!

**Pull request closed without merging**